lstchain/scripts/onsite/onsite_create_drs4_pedestal_file.py

```diff
@@ -80,10 +80,10 @@ def main():
 
     # update the default production directory
     if pro_symlink:
-        pro = "pro"
+        #pro = "pro"
```

Comment:
Please just remove instead of commenting out. Since the drs4 pedestal creation only needs raw data as input and not other calibration products, it is just not needed here.

Reply:
done